index.html

```diff
@@ -26,47 +42,70 @@ <h2>Add New Rate</h2>
           </form>
```

Comment:
First, prefer adding your event listener in JS using `addEventListener` instead of HTML attributes. It is easier to follow the logic when everything is centralised inside JS files.
Secondly, prefer a `<button type="submit">` since you are inside a `<form>` element.
Finally, you should use the `onsubmit` event from the `form` element instead of the `click` event of the button. If you press "Enter" while your mouse is in an input, it submits the form without inserting the new rate. Using the `onsubmit` event will solve that.

This comment is valid for the two other forms.